src/transport.js

```diff
@@ -0,0 +1,44 @@
+const nodemailer = require('nodemailer')
+const ses = require('nodemailer-ses-transport')
+const sendgrid = require('nodemailer-sendgrid-transport')
+const postmark = require('nodemailer-postmark-transport')
+const mailgun = require('nodemailer-mailgun-transport')
+const stub = require('nodemailer-stub-transport')
+const winston = require('winston')
+
+module.exports = env => {
+  switch(env('TRANSPORT')) {
+    case 'ses':
+      AWS.config.update({
+        accessKeyId: env('AWS_KEY'),
+        secretAccessKey: env('AWS_SECRET'),
+        region: env('AWS_REGION')
+      })
+      return nodemailer.createTransport(ses({ ses: new AWS.SES() }))
+    case 'sendgrid':
+      return nodemailer.createTransport(sendgrid({
+        auth: {
+          api_user: env('SENDGRID_USERNAME'),
+          api_key: env('SENDGRID_PASSWORD') || env('SENDGRID_API_KEY')
```

Comment:
Is this just allowing the user to write the env variable name in two different ways or is it that sendgrid allows two different values (password or api key)? In any case I would prefer just using the `SENDGRID_API_KEY`. I think using user passwords for APIs is an anti-pattern.